pkgs/by-name/heads/package.nix

```diff
@@ -0,0 +1,432 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchgit,
+  fetchurl,
+  newScope,
+  replaceVars,
+  runCommandNoCC,
+  unstableGitUpdater,
+  writeShellApplication,
+  _experimental-update-script-combinators,
+  autoconf,
+  bashNonInteractive,
+  bc,
+  binwalk,
+  bison,
+  ccache,
+  cmake,
+  cpio,
+  curl,
+  elfutils,
+  envsubst,
+  flex,
+  getopt,
+  gnat,
+  gnat-bootstrap,
+  gnum4,
+  gnumake,
+  git,
+  imagemagick,
+  innoextract,
+  jq,
+  ncurses,
+  nix,
+  nixfmt-rfc-style,
+  nix-prefetch-scripts,
+  perl,
+  pkg-config,
+  python3,
+  rsync,
+  uefi-firmware-parser,
+  unzip,
+  xz,
+  zip,
+  zlib,
+  brandName ? "Heads",
+}:
+
+let
+  # Theoretically, it should be possible to extract these in the updateScript by running the corresponding download
+  # scripts in /blobs/* with a curl/wget that prints out the given URL, and collecting all the requested URLs & their
+  # output names
+  blobs = [
```

Comment:
Note: I am *unhappy* to announce that while `deps.nix` is still 100% generated, the new `blobs` attrset in `package.nix` likely belongs in there logically… It was just easier to extract those few downloads with their peculiarities by hand than to hack together an automatable solution for now.

Unlike the *mountain of dependencies* that is `deps.nix`, manually updating these by skimming the scripts in `/blobs/*` should at least be fairly doable.